hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmMultipartUploadList.java

```diff
@@ -18,15 +18,21 @@
 
 package org.apache.hadoop.ozone.om.helpers;
 
-import java.util.ArrayList;
 import java.util.List;
 
+import org.apache.hadoop.hdds.client.ReplicationType;
+import org.apache.hadoop.hdds.protocol.proto.HddsProtos.ReplicationFactor;
+
 /**
- * List of in-flight MU upoads.
+ * List of in-flight MPU uploads.
  */
 public class OmMultipartUploadList {
 
-  private List<OmMultipartUpload> uploads = new ArrayList<>();
+  private ReplicationType replicationType;
+
+  private ReplicationFactor replicationFactor;
```

Comment:
Why do we need this in the multipart list result?
These are specific to each in-progress MPU.

Reply:
Yes, it was added during one of the experiments. I removed it.